src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/FeatureToggles/Feature.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.EAS.Domain.Models.FeatureToggles
+{
+    public class Feature
+    {
+        public FeatureType FeatureType { get; set; }
+
+        public string Name { get; set; }
+
+        public bool Enabled { get; set; }
```

Comment:
Is this property used?

Reply:
No, it wasn't but it is now used. A new operation authorisation handler has been added that checks this property before any other checks are made.